src/chainstate/stacks/miner.rs

```diff
@@ -140,6 +149,28 @@ pub fn signal_mining_ready(miner_status: Arc<Mutex<MinerStatus>>) {
     }
 }
 
+/// get the mining amount
+pub fn get_mining_spend_amount(miner_status: Arc<Mutex<MinerStatus>>) -> u64 {
+    match miner_status.lock() {
+        Ok(status) => status.get_spend_amount(),
+        Err(_e) => {
+            panic!("FATAL: mutex poisoned");
+        }
+    }
+}
+
+/// set the mining amount
+pub fn set_mining_spend_amount(miner_status: Arc<Mutex<MinerStatus>>, amt: u64) {
+    match miner_status.lock() {
```

Comment:
nit:

```rust
    miner_status
        .lock()
        .expect("FATAL: mutex poisoned")
        .set_spend_amount(amt);
```

Reply:
See this PR: https://github.com/stacks-network/stacks-blockchain/pull/3516

Reply:
Yup, will accept via #3516